common/djangoapps/student/models.py

```diff
@@ -929,12 +929,15 @@ def is_course_full(self, course):
 
         return is_course_full
 
-    def users_enrolled_in(self, course_id):
-        """Return a queryset of User for every user enrolled in the course."""
-        return User.objects.filter(
-            courseenrollment__course_id=course_id,
-            courseenrollment__is_active=True
-        )
+    def users_enrolled_in(self, course_id, include_inactive_users=False):
+        """Return a queryset of User for every user enrolled in the course.  If `include_inactive_users` is True,
+        returns both active and inactive enrollees for the course. Otherwise returns actively enrolled users only."""
```

Comment:
Nit: Our formatting style is to use multi-line docstrings.  See the [edX python guidelines](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings).

Also, although not enforced and not followed throughout the codebase, keeping comments/docstrings to 80 characters is good citizenship for certain editors.  See https://www.python.org/dev/peps/pep-0008/#maximum-line-length   Code lines can be at the 120 character limit.